content/en/llm_observability/quickstart.md

```diff
@@ -129,6 +129,75 @@ The trace you see is composed of a single LLM span. The `ddtrace-run` or `NODE_O
 
 If your application consists of more elaborate prompting or complex chains or workflows involving LLMs, you can trace it using the [Setup documentation][11] and the [SDK documentation][1].
 
+## Trace an LLM application in AWS Lambda
+This generates an LLM Observability trace in an AWS Lambda environment and creates an Amazon Bedrock based chatbot running with LLM Observability in AWS Lambda.
+
+1. Create a [Lambda function chatbot using Amazon Bedrock][13].
+2. Instrument your Lambda function:
+    1. Open a Cloudshell
+    2. Install the Datadog CLI client
+    ```shell
+    npm install -g @datadog/datadog-ci
+    ```
+    3. Set the Datadog API key and site
+    ```shell
+    export DD_SITE=<YOUR_DD_SITE>
+    export DD_API_KEY=<YOUR_DATADOG_API_KEY>
+    ```
+    If you already have or prefer to use a secret in Secrets Manager, you can set the API key by using the secret ARN:
+    ```shell
+    export DATADOG_API_KEY_SECRET_ARN=<DATADOG_API_KEY_SECRET_ARN>
+    ```
+    4. Instrument your Lambda function with LLM Observability (this requires at least version 77 of the Datadog Extension layer).
+{{< tabs >}}
```

Comment:
Because this tabbed section is nested inside a numbered list, the tabs are indented. I think it looks a bit odd, but perhaps it is intentional and desired? Take a look and tell me what you think. If you'd like the tabs to be left justified, I think adding some extra newlines will separate them from the numbered list.

To see what I mean about the indentation, go to the [preview](https://docs-staging.datadoghq.com/sabrenner/llmobs-serverless-quickstart/llm_observability/quickstart/?tab=python), search on the page for the string "77", and look at the tabs just below the search result.

Reply:
ahh yes i see what you're saying! yeah let me try adding some newlines around it to see if it helps a bit. it's not necessarily intentional, but otherwise i don't think it's something we're too concerned about.

Reply:
I also played around with newlines. I was able to get the tab left justified, but it looked strange. I think the current configuration is the best. Thanks for being willing to experiment!